### ISSUE TITLE
[vscode-graphql-syntax] Problem with GraphQL Syntax Highlighting

### ISSUE THREAD

**Issue by B2o5T** [graphql/graphiql#3217]
### Is there an existing issue for this?

- [X] I have searched the existing issues

**Comment by acao** [comment]
yes this is where the issue would be. we badly need to add tests for the syntax extension, because minor improvements sometimes cause regressions, or we overlook lots of edge cases like these. i can pull up the vscode docs for writing tests for the grammar configs

**Comment by B2o5T** [comment]
Yeah! I would like to add image snapshots tests before fixing some issues
